src/pages/RoomInvitePage.tsx

```diff
@@ -222,6 +236,7 @@ function RoomInvitePage({betas, report, policies}: RoomInvitePageProps) {
                     showScrollIndicator
                     shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
                     showLoadingPlaceholder={!areOptionsInitialized}
+                    isLoadingNewOptions={!!isSearchingForReports}
```

Comment:
Why are we using a double exclamation mark here? I thought the value of `isSearchingForReports` would always be boolean.

Reply:
I thought that too, but apparently not.
Even in `SearchPage` (I didn't change anything there because it was already working) we use this check:
https://github.com/Expensify/App/blob/a7820d2027138a1a9670b204f608d6b92ec0a454/src/pages/SearchPage/index.tsx#L187
I thought I'd add the 'double exclamation mark' in `SearchPage` too. What do you think?

Reply:
Done :D